go-controller/pkg/config/config.go

```diff
@@ -447,9 +448,11 @@ type GatewayConfig struct {
 	V4JoinSubnet string `gcfg:"v4-join-subnet"`
 	// V6JoinSubnet to be used in the cluster
 	V6JoinSubnet string `gcfg:"v6-join-subnet"`
-	// V4MasqueradeSubnet to be used in the cluster
+	// V4MasqueradeSubnet to be used in the cluster has to be bigger than
+	// MaxUserDefinedNetworks * 2 + 5
```

Comment:
plus 10 I'd say?

Reply:
cc @arghosh93 we need some validation around this when we accept new values whether its enough or not
so default should be made `169.254.169.0/16` which gives us 65,534 IPs to use which is greater than 60010 that we need.

Reply:
yes @tssurya , absolutely.  Are we going with MaxUserDefinedNetworks * 2 + 5 or MaxUserDefinedNetworks * 2 + 10.

Reply:
Yep, this is a typo, we have mark the MasqueradeIPBase as 10.